doc/source/cluster/kubernetes/user-guides/ack-gpu-cluster.md

```diff
@@ -0,0 +1,28 @@
+(kuberay-ack-gpu-cluster-setup)=
```

Comment:
I don't have access to Aliyun, so I can't verify whether this doc is easy to follow or not. Would you mind asking one of your colleagues to review this document?

Reply:
sure, thanks for suggestion